app/app/test/page.jsx

```diff
@@ -1,87 +1,92 @@
+/* eslint-disable @next/next/no-img-element */
```

Comment:
Good try :) 
In case if you want to disable some rule there is several ways to do so:
- discuss with the team and change this rule for everyone
- write a disable comment (which you did) AND write the comment why it should be disabled in this particular case next to it

Anyway, rule about images is pretty good: it doesn't make sense to use not-optimized component for images when you can have good image performance out of the box